core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -190,6 +177,37 @@ private void createSchemaTopic() throws StoreInitializationException {
     }
   }
 
+  private String getBrokerEndpointsFromZk(Seq<Broker> brokers) {
```

Comment:
- maybe update name since this isn't actually connecting to zookeeper?
- also it'd be good to add a unit test for this method; probably reasonable to make it package private so it's available to test.